assets/app/views/directives/logs/_log-viewer.html

```diff
@@ -6,7 +6,10 @@
   style="margin-bottom: 10px;">
   <div ng-transclude flex><!-- expect start time, end time, status details --></div>
   <div row class="text-right text-xs-left">
-    <a href="" ng-click="goChromeless(options)" style="margin-right: 10px;">
+    <a ng-if="state !== 'empty'"
```

Comment:
Yeah, I didn't want to expose `state` outside the directive. I'm not crazy about the fix.

Right now, only the parent knows about `empty` and the log viewer uses `state` internally.